pkg/lclient/images.go

```diff
@@ -21,7 +21,7 @@ func (dc *Client) PullImage(image string) error {
 	}
 	defer out.Close()
 
-	if klog.V(4) {
+	if klog.V(1).{
```

Comment:
You forgot to remove the `.` in `if klog.V(1).{` thus the build is failing
```
# github.com/openshift/odo/pkg/lclient
pkg/lclient/images.go:24:15: syntax error: unexpected {, expecting name or (
pkg/lclient/images.go:25:35: syntax error: unexpected ), expecting {
pkg/lclient/images.go:26:3: syntax error: unexpected if, expecting { after if clause
# github.com/openshift/odo/pkg/lclient
vet: pkg/lclient/images.go:24:15: expected selector or type assertion, found '{' (and 3 more errors)
Makefile:86: recipe for target 'vet' failed
make: *** [vet] Error 2
The command "make validate" exited with 2.
```

Reply:
Woops!